rust/src/gildedrose.rs

```diff
@@ -21,10 +23,26 @@ impl Display for Item {
     }
 }
 
+#[derive(Debug)]
 pub struct GildedRose {
     pub items: Vec<Item>,
 }
 
+impl PartialEq for GildedRose {
```

Comment:
This is implementing a trait on the GildedRose to define how to compare? V. nice. I have not even looked at traits yet.